ui/packages/catalog-ui-search/src/main/webapp/component/input/enum/input-enum.less

```diff
@@ -1,4 +1,8 @@
 @{customElementNamespace}input.is-enum {
+  &.is-editing {
```

Comment:
Does this cause issues in other areas, such as the metacard inspector?

It might be better to target this component only in the lightbox.result.less so styling isn't affected elsewhere.